src/modules/Lan.rb

```diff
@@ -1013,6 +1013,20 @@ def HaveXenBridge
       have_br
     end
 
+    # Provides a list with the NTP servers obtained via any of dhcp aware
+    # interfaces
+    #
+    # @note parsing dhcp ntp servers when NetworkManager is in use is not
+    #   supported yet (bsc#798886)
+    #
+    # @return [Array<String>] list of ntp servers obtained byg DHCP
+    def dhcp_ntp_servers
+      return [] if !NetworkService.isNetworkRunning || Yast::NetworkService.is_network_manager
+
+      ReadWithCacheNoGUI()
+      Yast::LanItems.dhcp_ntp_servers.values.reduce(&:concat) || []
```

Comment:
NP: why not `...values.flatten` instead of `...values.reduce(&:concat)`?

Reply:
I originally just moved the method from where it was implemented, and I'm fine with both and probably would even verify a uniqueness.